onebusaway-android/src/main/java/org/onebusaway/android/ui/HomeActivity.java

```diff
@@ -2001,4 +2008,22 @@ private boolean isSlidingPanelCollapsed() {
     public ArrivalsListFragment getArrivalsListFragment() {
         return mArrivalsListFragment;
     }
+
+    private void checkBatteryOptimizations() {
+        if (Application.isIgnoringBatteryOptimizations(getApplicationContext())) {
```

Comment:
Shouldn't this be:

`if (!Application.isIgnoringBatteryOptimizations(getApplicationContext())) {`

We want the app to be ignoring battery optimizations.